acq_stat_reports/scripts/acq_stat_reports.py

```diff
@@ -0,0 +1,140 @@
+    )
+    parser.add_argument(
+        "--url",
+        help="URL root of the deployed page (default is /mta/ASPECT/acq_stat_reports/)",
+        default="/mta/ASPECT/acq_stat_reports/",
+    )
+    parser.add_argument(
+        "--start_time", help="Start time (default is NOW)", default=None
+    )
+    parser.add_argument(
+        "--days_back",
+        help="How many days to look back (default is 30)",
+        default=30,
+        type=int,
+    )
+    parser.add_argument(
+        "--log-level",
+        default="INFO",
+        choices=[
+            "DEBUG",
+            "INFO",
+            "WARNING",
+            "ERROR",
+            "CRITICAL",
+            "debug",
+            "info",
+            "warning",
+            "error",
+            "critical",
+        ],
+        help="Verbosity (DEBUG, INFO, WARNING, ERROR, CRITICAL)",
+    )
+    return parser
+
+
+def main(sys_args=None):
+    import matplotlib
+
+    matplotlib.use("agg")
+
+    args = get_parser().parse_args(sys_args)
+
+    logger.setLevel(args.v.upper())
+
+    now = CxoTime()
+    logger.info("---------- acq stat reports update at %s ----------" % (now.iso))
+
+    if args.start_time is None:
+        to_update = ska_report_ranges.get_update_ranges(args.days_back)
+    else:
+        start = CxoTime(args.start_time)
+        to_update = ska_report_ranges.get_update_ranges(
+            int((now - start).to(u.day).value)
+        )
+
+    all_acq = get_data()
+
+    for tname in sorted(to_update.keys()):
+        logger.debug("Attempting to update %s" % tname)
+        range_datestart = CxoTime(to_update[tname]["start"])
+        range_datestop = CxoTime(to_update[tname]["stop"])
+
+        # ignore acquisition stars that are newer than the end of the range
+        # in question (happens during reprocessing) for consistency
+        all_acq_upto = all_acq[all_acq["tstart"] <= CxoTime(range_datestop).secs]
+
+        output_dir = (
+            args.output_dir / f"{to_update[tname]['year']}" / to_update[tname]["subid"]
+        )
+        output_dir.mkdir(parents=True, exist_ok=True)
+
+        dataout = (
+            args.datadir / f"{to_update[tname]['year']}" / to_update[tname]["subid"]
+        )
+        dataout.mkdir(parents=True, exist_ok=True)
+
+        logger.debug("Plots and HTML to %s" % output_dir)
+        logger.debug("JSON to  %s" % dataout)
+
+        acq_info = get_acq_info(all_acq_upto, tname, range_datestart, range_datestop)
+        with open(dataout / "acq_info.json", "w") as rep_file:
+            rep_file.write(json.dumps(acq_info, sort_keys=True, indent=4))
+
+        prev_range = ska_report_ranges.get_prev(to_update[tname])
+        next_range = ska_report_ranges.get_next(to_update[tname])
+        nav = {
+            "main": args.url,
+            "next": f"{args.url}/{next_range['year']}/{next_range['subid']}/index.html",
+            "prev": f"{args.url}/{prev_range['year']}/{prev_range['subid']}/index.html",
+        }
```

Comment:
Why the absolute URLs here instead of relative? This caused issues for me testing and requires matching `--url` to the `--webdir` option.

Reply:
This is all gone now. There is a single page, so no need for navigation, url, or anything.